apps/prairielearn/src/ee/pages/studentCourseInstanceUpgrade/studentCourseInstanceUpgrade.html.ts

```diff
@@ -56,8 +56,9 @@ export function StudentCourseInstanceUpgrade({
             requires an upgrade to support certain features selected by your instructor.
           </p>
 
-          ${planPrices !== null
-            ? html`
+          ${planPrices == null
+            ? html`<p>Please contact your instructor for more information.</p>`
```

Comment:
This is to ensure that we show something sensible in case Stripe isn't configured on an enterprise instance.